week-2/errors/0.js

```diff
@@ -3,7 +3,10 @@
 // then call the function capitalise with a string input
 // interpret the error message and figure out why it's happening, if your prediction was wrong
 
+
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+ let str = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
-}
+} 
+
+// str is defined two times, once in function parameter and again in function body, to chnage the error, all we hve to do is remove let from let str and it will work 
```

Comment:
You have correctly identified the problem with the code.
The aim of this section is to get familiar with [error messages in Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors). To that end, what error message do you think will come up when you run this code? What error message was actually displayed?

Reply:
the error message was idetifier str has been already been declared. So to solve that you can do what I described in the comments